<!doctype html>
<html lang="en">
  <head>
    <meta charset="utf-8">
    <meta name="viewport" content="width=device-width, initial-scale=1">
    <title>D. Taliun's Lab</title>
    <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.2.1/dist/css/bootstrap.min.css" rel="stylesheet" integrity="sha384-iYQeCzEYFbKjA/T2uDLTpkwGzCiq6soy8tYaI1GyVh/UjpbCx/TYkiZhlZB6+fzT" crossorigin="anonymous">
  </head>
  <body data-bs-spy="scroll" data-bs-target=".navbar" data-bs-offset="40">
    <nav class="navbar navbar-expand-sm bg-light navbar-light fixed-top">
        <div class="container-fluid">
            <a class="navbar-brand" href="#">
                <img src="bioinformatics.png" alt="Avatar Logo" style="width:40px;" class="rounded-pill">
            </a>
            <button class="navbar-toggler" type="button" data-bs-toggle="collapse" data-bs-target="#collabsibleNavbar">
                <span class="navbar-toggler-icon"></span>
            </button>
            <div class="collapse navbar-collapse" id="collabsibleNavbar">
                <ul class="navbar-nav">
                    <li class="nav-item">
                         <a class="nav-link" href="#about">About</a>
                    </li>
                    <li class="nav-item">
                         <a class="nav-link" href="#projects">Projects</a>
                    </li>
                    <li class="nav-item">
                         <a class="nav-link" href="#positions">Positions</a>
                    </li>
                </ul>
            </div>
        </div>
    </nav>
      
    <div id="about" class="container-fluid text-center" style="padding:80px 20px;">
        <div class='row'>
            <div class="col">
                <p class="h1">About</p>
                <hr/>
                <p>My name is Daniel Taliun, and I am an Assistant Professor in the <a href="https://www.mcgill.ca/humangenetics/">Department of Human Genetics</a> at McGill University in <a href="https://en.wikipedia.org/wiki/Montreal">Montreal, Canada</a>. I am also a member of the <a href="https://www.genomic-medicine-cerc.online/">McGill CERC Program in Genomic Medicine</a>, where I lead research efforts in computational genomics. I did my Ph.D. in Computer Science, developing computational algorithms and analytical tools for <a href="https://en.wikipedia.org/wiki/Genome-wide_association_study">genetic association studies of single-nucleotide polymorphisms</a>. As genetic technologies improve, I further expand my research interests to meet new challenges in data analysis raised by new types of genetic data and genetic experiments. What excites me about my work is the data variety and size and the <a href="https://www.nature.com/articles/456018a">unsolved puzzles</a> of how our genes impact our health. My goal is to pass on all my experience and excitement about this field to all trainees working in my team.</p>
            </div>
        </div>
    </div>
      
    <div id="projects" class="container-fluid text-center" style="padding:80px 20px;">
        <div class='row'>
            <div class="col">
                <p class="h1">Projects</p>
                <hr/>
                <table class="table table-hover table-sm text-start">
                    <thead>
                        <tr>
                            <th scope="col">Project</th>
                            <th scope="col">Description</th>
                        </tr>
                    </thead>
                    <tbody>
                        <tr>
                            <td>HLA diversity and role in disease</td>
                            <td>We investigate genetic variations in <a href="https://en.wikipedia.org/wiki/Human_leukocyte_antigen">human leukocyte antigen</a> (HLA) genes on chromosome 6, which are responsible for regulating our immune system. We develop methods and tools for detecting variations in HLA using <a href="https://en.wikipedia.org/wiki/DNA_sequencing">next-generation sequencing</a> methods and apply them to data from Canadian genetic studies. We develop methods and tools for the <a href="https://en.wikipedia.org/wiki/Imputation_(genetics)">imputation</a> of missing HLA variants. We perform statistical association analyses of variations in HLA with various diseases and disease-related traits.
                            </td>
                        </tr>
                        <tr>
                            <td>SNVs and InDels diversity and role in disease</td>
                            <td>We develop tools for detecting rare <a href="https://en.wikipedia.org/wiki/Single-nucleotide_polymorphism">single-nucleotide variations</a> (SNVs) and <a href="https://en.wikipedia.org/wiki/Indel">insertions/deletions</a> (InDels) using <a href="https://en.wikipedia.org/wiki/DNA_sequencing">next-generation sequencing</a> data and apply them to Canadian genetic studies. We develop methods and tools for the <a href="https://en.wikipedia.org/wiki/Imputation_(genetics)">imputation</a> of SNVs/InDels. We perform statistical association analyses of SNVs and InDels with diseases and disease-related traits.</td>
                        </tr>
                        <tr>
                            <td>SV diversity and role in disease</td>
                            <td>We develop tools for detecting <a href="https://en.wikipedia.org/wiki/Structural_variation">structural variations</a> (SV) using <a href="https://en.wikipedia.org/wiki/DNA_sequencing">next-generation sequencing</a> data and apply them to Canadian genetic studies. We develop methods and tools for the <a href="https://en.wikipedia.org/wiki/Imputation_(genetics)">imputation</a> of structural variations. We perform association mapping of structural variations with diseases and disease-related traits.</td>
                        </tr>
                        <tr>
                            <td>GWAS and PheWAS in large genetic studies and biobanks</td>
                            <td><a href="https://en.wikipedia.org/wiki/Genome-wide_association_study">Genome-wide association scans</a> and <a href="https://en.wikipedia.org/wiki/Phenome-wide_association_study">phenome-wide association scans</a> across thousands of diseases and disease-related traits in large Canadian genetic studies, including <a href="https://www.clsa-elcv.ca/">Canadian Longitudinal Study of Aging (CLSA)</a> and <a href="https://www.cartagene.qc.ca/">CARTaGENE</a>.</td>
                        </tr>
                        <tr>
                            <td>Transferability of PRS</td>
                            <td>This project explores if <a href="https://en.wikipedia.org/wiki/Polygenic_score">polygenic risk scores</a> (PRS) developed using international/external biobank data (e.g. <a href="https://en.wikipedia.org/wiki/UK_Biobank">UK Biobank</a>) can be applied to Canadian populations. Using external biobank data, we will evaluate the pros/cons of different strategies to build the most precise PRS for Canadian people. The project will result in novel statistical approaches and computation tools to support the research community in Canada</td>
                        </tr>
                        <tr>
                            <td>Statistical and ML methods for including of SV into PRS</td>
                            <td>Most existing methods for computing <a href="https://en.wikipedia.org/wiki/Polygenic_score">PRS</a> use <a href="https://en.wikipedia.org/wiki/Single-nucleotide_polymorphism">SNVs</a> and <a href="https://en.wikipedia.org/wiki/Indel">InDels</a>. This project will develop strategies to include <a href="https://en.wikipedia.org/wiki/Structural_variation">SVs</a> in the computation of PRS and assess the change in PRS predicitive power.</td>
                        </tr>
                        <tr>
                            <td>Statistical methods for the X chromosome association mapping</td>
                            <td>The current statistical methods for gene-disease associations and association mapping in chromosome X follow the same approach as in autosomal chromosomes. They remain blind to unique chromosome X <a href="https://en.wikipedia.org/wiki/X-linked_recessive_inheritance">inheritance patterns</a> and <a href="https://www.nature.com/scitable/topicpage/x-chromosome-x-inactivation-323/">X-inactivation</a> mechanisms. This project aims to improve existing and develop new statistical methods and tools for gene-disease association mapping on chromosome X and evaluate them on the largest biobanks.</td>
                        </tr>
                        <tr>
                            <td>Statistical methods for genotype meta-imputation using low-coverage WGS</td>
                            <td>This project aims to develop a meta-imputation strategy for <a href="https://en.wikipedia.org/wiki/Coverage_(genetics)">low-coverage</a> <a href="https://en.wikipedia.org/wiki/Whole_genome_sequencing">whole-genome sequencing</a> (WGS) data. The meta-imputation is a novel statistical approach that combines <a href="https://en.wikipedia.org/wiki/Imputation_(genetics)">genotype imputation</a> results from multiple reference panels, thus, significantly improving the imputation accuracy of population-specific genetic variants. However, the current methods support only tools developed for the imputation using genotyping <a href="https://en.wikipedia.org/wiki/SNP_array">array-based</a> data and do not support low-coverage WGS data.</td>
                        </tr>
                        <tr>
                            <td>Statistical and ML-based genotype imputation approaches using fused WGS and WES reference panels</td>
                            <td>Most current methods for <a href="https://en.wikipedia.org/wiki/Imputation_(genetics)">imputing missing genotypes</a> use reference panels derived from <a href="https://en.wikipedia.org/wiki/Whole_genome_sequencing">whole-genome sequencing</a> (WGS) data. The vast amounts of available <a href="https://en.wikipedia.org/wiki/Exome_sequencing">whole-exome sequencing</a> (WES) data could improve the imputation quality of rare protein-coding variants but remain unused for imputation purposes. In this project, we try to develop solutions that would use WES and WGS data together to improve genotype imputation qualities.</td>
                        </tr>
                        <tr>
                            <td>Web-based server for genotype imputation</td>
                            <td>Many genetic analyses are computationally demanding and require access to high-performance compute clusters and large datasets. We are developing a series of automated web-based services to facilitate researchers in such genetic analyses. This project aims to build a secure web-based server and accompanying computational tools to enable accurate <a href="https://en.wikipedia.org/wiki/Imputation_(genetics)">genotype imputation</a> in Canadian populations using Canada-specific reference panels in combination with state-of-the-art global reference panels. Examples of such services are <a href="https://imputation.biodatacatalyst.nhlbi.nih.gov/#!">NHLBI TOPMed Imputation Server</a> and <a href="https://imputationserver.sph.umich.edu/index.html#!">Michigan Imputation Server</a>.</td>
                        </tr>
                        <tr>
                            <td>Web-based servers for genetic ancestry estimation</td>
                            <td>This project aims to build a secure web-based server and accompanying computational tools to enable accurate <a href="https://en.wikipedia.org/wiki/Genealogical_DNA_test">genetic ancestry</a> estimation in Canadian populations using Canada-specific reference panels.</td>
                        </tr>
                        <tr>
                            <td>Web-based servers for HLA allele imputation</td>
                            <td> This project aims to build a secure web-based server and accompanying computational tools to enable accurate <a href="https://en.wikipedia.org/wiki/Human_leukocyte_antigen">HLA</a> alleles <a href="https://en.wikipedia.org/wiki/Imputation_(genetics)">imputation</a> in Canadian populations using Canada-specific reference panels in combination with state-of-the-art global reference panels. </td>
                        </tr>

 

                    </tbody>
                </table>
            </div>
        </div>
    </div>
      
    <div id="positions" class="container-fluid text-center" style="padding:80px 20px;">
        <div class='row'>
            <div class="col">
                <p class="h1">Positions</p>
                <hr/>
                <p>I am seeking MSc and PhD students with a strong interest in computational work applied to human genetics. If you are interested, please fill out this <a href="CV_template.docx">CV template</a> and send it to daniel.taliun@mcgill.ca with the Subject line "Application to MSc or PhD".</p>
            </div>
        </div>
    </div>

      
<!--    <div class="container text-center">-->
<!--        <a href="https://www.flaticon.com/free-icons/bioinformatics" title="bioinformatics icons">Bioinformatics icons created by Parzivalâ€™ 1997 - Flaticon</a>-->
<!--        <a href="https://www.flaticon.com/free-icons/bioinformatics" title="bioinformatics icons">Bioinformatics icons created by Freepik - Flaticon</a>-->
<!--    </div>-->
    <script src="https://cdn.jsdelivr.net/npm/bootstrap@5.2.1/dist/js/bootstrap.bundle.min.js" integrity="sha384-u1OknCvxWvY5kfmNBILK2hRnQC3Pr17a+RTT6rIHI7NnikvbZlHgTPOOmMi466C8" crossorigin="anonymous"></script>
  </body>
</html>
